components/button/src/button/__tests__/Button.test.js

```diff
@@ -4,6 +4,52 @@ import React from 'react'
 import { Button } from '../button.js'
 
 describe('<Button>', () => {
+    let consoleSpy
+
+    beforeEach(() => {
+        consoleSpy = jest.spyOn(console, 'debug').mockImplementation()
+    })
+
+    afterEach(() => {
+        consoleSpy.mockRestore()
+    })
+
+    describe('warning for missing aria-label or title', () => {
+        it('No warning  if children exist but aria-label or title is missing', () => {
+            render(<Button>Children content</Button>)
+
+            expect(consoleSpy).not.toHaveBeenCalled()
+        })
+
+        it('does not warn if aria-label or title is present', () => {
```

Comment:
This "or" should be "and" according to the test and implementation, right?

Reply:
yeah, you're right, "and" seems like a better choice in this case.